py-polars/tests/unit/io/test_parquet.py

```diff
@@ -1255,6 +1255,9 @@ def test_parquet_list_element_field_name() -> None:
     assert "child 0, element: int64" in schema_str
 
 
+@pytest.mark.skip(
+    reason="TODO: fix Parquet writing Decimal-in-struct with precision > 18"
```

Comment:
opened an issue for this at https://github.com/pola-rs/polars/issues/19448 - this bug had always existed but we never caught it because `assert_frame_equal` used the incorrect eq_missing on structs